### PR TITLE
Support sort queries on metrics generated by eBPF receiver

### PR DESCRIPTION
**Pull request by mrproliu**
For now, the metrics generated by the eBPF receiver cannot be using `top_n` queries in MQE, so I updated the sources to support this feature. And delete the useless endpoint relation.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: docs/en/concepts-and-designs/scope-definitions.md [hunk 1]
```diff
@@ -529,9 +529,9 @@ following **package**/**protocol** level metric contents.
 | destServiceInstanceName   | The dest pod name in kubernetes.                                   |           | string |
 | destLayer                 | The dest layer service in kubernetes.                              |           | string |
 
-#### Endpoint and Endpoint Relation
+#### Endpoint
```
Comment (wu-sheng) [inline docs/en/concepts-and-designs/scope-definitions.md:532]: `DefaultScopeDefine#K8S_ENDPOINT_REALATION(ID=77)` should be removed.

File: oap-server/server-receiver-plugin/skywalking-ebpf-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/ebpf/provider/handler/AccessLogServiceHandler.java [hunk 2]
```diff
@@ -648,26 +647,6 @@ public K8SEndpoint toEndpoint(String endpointName, boolean success, long duratio
             return endpoint;
         }
 
-        public K8SEndpointRelation toEndpointRelation(String endpointName, boolean success) {
```
Comment (wu-sheng) [inline oap-server/server-receiver-plugin/skywalking-ebpf-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/ebpf/provider/handler/AccessLogServiceHandler.java:651]: What was this building? API between client side and server side?

**Review by wu-sheng — APPROVED** [review]

File: docs/en/concepts-and-designs/scope-definitions.md (hunk 1 above)
Comment (mrproliu) [reply docs/en/concepts-and-designs/scope-definitions.md:532]: Done.

File: oap-server/server-receiver-plugin/skywalking-ebpf-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/ebpf/provider/handler/AccessLogServiceHandler.java (hunk 2 above)
Comment (mrproliu) [reply oap-server/server-receiver-plugin/skywalking-ebpf-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/ebpf/provider/handler/AccessLogServiceHandler.java:651]: Yes.

**Pull request merged**